sdcm/cluster_aws.py

```diff
@@ -351,6 +351,21 @@ def network_config_ipv6_workaround_script():  # pylint: disable=invalid-name
             IPv6_CIDR=`curl -s ${BASE_EC2_NETWORK_URL}${MAC}/subnet-ipv6-cidr-blocks`
 
             grep -qi "amazon linux" /etc/os-release || sudo ip route add $IPv6_CIDR dev eth0
+
+            grep -q IPV6_AUTOCONF /etc/sysconfig/network-scripts/ifcfg-eth0
```

Comment:
also, in bash there is a notation of `&&` and `||`
so it could be simplified to (although it think this way is much clearer):
```
( grep -q IPV6_AUTOCONF /etc/sysconfig/network-scripts/ifcfg-eth0 && sudo echo "IPV6_AUTOCONF=yes" >> /etc/sysconfig/network-scripts/ifcfg-eth0 ) || sudo sed -i 's/^IPV6_AUTOCONF=[^ ]*/IPV6_AUTOCONF=yes/' /etc/sysconfig/network-scripts/ifcfg-eth0
```
but of course the syntax could have to be adjusted if you decide to move it...
if you are done, i will approve it, as it LGTM

Reply:
I didn't try to write in one line. I checked that this code version works :)
If you insist I can try to change to one line syntax

Reply:
no no, LGTM